docs/specs/supporting-tools.md

```diff
@@ -0,0 +1,72 @@
+# Supporting tools and services
+
+This page outlines tools and services that currently support the development container specification, including the `devcontainer.json` format. A `devcontainer.json` file in your project tells tools and services that support the dev container spec how to access (or create) a dev container with a well-defined tool and runtime stack. 
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## Editors
+
+### Visual Studio Code
+
+Visual studio code specific properties go under `vscode` inside `customizations`.
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `extensions` | array | An array of extension IDs that specify the extensions that should be installed inside the container when it is created. Defaults to `[]`. |
+| `settings` | object | Adds default `settings.json` values into a container/machine specific settings file. Defaults to `{}`. |
+
+Please note that Codespaces supports the VS Code properties.
+
+## Tools
```

Comment:
Reorganized this section to make it easier in the future to add other supporting tools and editors and the possible combinations thereof. Comments? @Chuxel @bamurtaugh

Reply:
I like the idea of separating based on editors vs other tools, but I was a bit confused by how the product specifics are listed. VS Code is in its own section at the top, but the Remote - Containers section is at the bottom, and both have their own product specific properties.

Upon re-reading, I understand it better, but I wonder if the Remote - Containers extension content should just be contained within the VS Code section. I suppose there could be confusion as the VS Code properties apply to both Remote - Containers and desktop Codespaces?

Reply:
Yeah the idea is that this settings are supported by both and might not be supported by other tools, when we want to add other editors that might only be supported by some tools.

Made an update let me know if you like the wording.

Reply:
I like it, thanks for updating!